specification/storage/resource-manager/Microsoft.Storage/stable/2019-04-01/storage.json

```diff
@@ -1628,6 +1628,25 @@
           "x-ms-client-name": "FailoverInProgress",
           "description": "If the failover is in progress, the value will be true, otherwise, it will be null.",
           "readOnly": true
+        },
+        "minimumTLSVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "TLS version control on storage account in customer configuration, TLS v1.2 by default.",
```

Comment:
If it's both read-only and defaults to v1.2, it is impossible to ever get v1.0 and v1.1, right?